lua/net.lua

```diff
@@ -20,7 +20,7 @@ function M.setup_block()
 
 	-- Change these parameters to configure the network.
 	local front_ports = {"enp133s0f0"}
-	local front_rx_queues = 2
+	local front_rx_queues = 3
```

Comment:
Do you need to change the ``back_tx_queues``? Since the GGU only receives packets from the GT, and send the decisions to GK using mailbox.